deploy/cdk/marble-elasticsearch-pipeline/repo.json

```diff
@@ -0,0 +1,5 @@
+{
+    "repo_name": "marble-elasticsearch",
+    "repo_owner": "ndlib",
+    "repo_branch": "es-domain"
```

Comment:
should this repo_branch be "master"?

Reply:
Yes that's probably a better fit. I went and updated it. Thanks